HUD/N_0xf83d0febe75e62c9.md

```diff
@@ -5,20 +5,33 @@ ns: HUD
 
 ```c
 // 0xF83D0FEBE75E62C9
-void _0xF83D0FEBE75E62C9(Any p0, Any p1, Any p2, Any p3, Any p4, Any p5, Any p6, Any p7);
+void _0xF83D0FEBE75E62C9(cs_type(Any) Blip blip, Any p1, Any p2, cs_type(Any) float width, Any p4, cs_type(Any) float length, cs_type(Any) float heading, Any p7);
 ```
 
 ```
-NativeDB Added Parameter 9: Any p8
+NativeDB Added Parameter 9: int hudColour
 NativeDB Introduced: v1290
 ```
 
+Sets the 'data' for a cone for a blip
+
+See [here](https://docs.fivem.net/docs/game-references/hud-colors/) for the hud colours for the cone
```

Comment:
Mixed British and American spelling (color/colour), do we apply a standard for docs? (not method signatures)

Reply:
> Mixed British and American spelling (color/colour), do we apply a standard for docs? (not method signatures)

I'm not really sure, even native names seem to mix these😅
I prefer the AmE spelling though, searched for it and found more occurrences.

Reply:
Ah yeah with method signatures I was referring to the natives, seems like R* is also not sure which one to pick, lol.
I also prefer AmE, but more so sticking to one or the other :)